systori/static/ts/src/project_list.ts

```diff
@@ -149,97 +60,66 @@ class SystoriSortButton extends HTMLElement {
             ? (this.dataset["reversed"] = "true")
             : (this.dataset["reversed"] = "false");
     }
+    toggleReversed(): void {
+        this.reversed = !this.reversed;
+    }
 
     get active(): boolean {
         return this.classList.contains("active");
     }
     set active(active: boolean) {
         active ? this.classList.add("active") : this.classList.remove("active");
     }
+    sortProjectTiles(this: SystoriSortButton): void {
```

Comment:
This is a class method no need for `this` parameter, adds more confusion.
```suggestion
    sortProjectTiles(): void {
```